libiocage/lib/JailConfig.py

```diff
@@ -607,7 +537,7 @@ def set(self, key: str, value, **kwargs) -> bool:
         return (hash_before != hash_after)
 
     def __str__(self):
```

Comment:
Missing return annotation

Reply:
Ha, that is obvious! Of course it returns an integer!!